exareme2/worker/flower/starter/starter_service.py

```diff
@@ -57,11 +68,19 @@ def start_flower_server(
         "CONTROLLER_IP": worker_config.controller.ip,
         "CONTROLLER_PORT": worker_config.controller.port,
         "DATA_PATH": worker_config.data_path,
-        "CSV_PATHS": ",".join(csv_paths),
+        "CSV_PATHS": get_absolute_dataset_csv_paths(data_model, datasets),
```

Comment:
In the future we should pass all of these, inputdata AND env variables in an sqlite table. I will make a ticket.